examples/farming_sim.rs

```diff
@@ -336,22 +336,20 @@ pub fn sell_need_scorer_system(
 
 // This is a component that will be attached to the actor entity when it is
 // moving to a location. It's not an AI component, but a standard Bevy component
-#[derive(Clone, Component, Debug)]
+#[derive(Debug, Clone, Component, ActionBuilder)]
+#[action_label = "MyGenericLabel"]
 pub struct MoveToNearest<T: Component + std::fmt::Debug + Clone> {
-    _marker: std::marker::PhantomData<T>,
-    speed: f32,
+    // We use a PhantomData to store the type of the component we're moving to.
+    pub _marker: std::marker::PhantomData<T>,
+    pub speed: f32,
```

Comment:
these don't need to be public and, in fact, you usually don't want `PhantomData` to be `pub`.

Reply:
done